packages/client/src/components/ProfilePage/ProfilePage.tsx

```diff
@@ -1,17 +1,26 @@
+import { Button } from '@mui/material';
 import React from 'react';
 
 import { useStyles } from './useStyles';
 
+import useAuthController from '../../services/controllers/useAuthController';
+import { useAppSelector } from '../../store/hooks';
+import { InitialUserType } from '../../store/interfaces';
 import Layout from '../Layout/Layout';
 
 const ProfilePage = () => {
   const classes = useStyles();
+  const { first_name, second_name } = useAppSelector<InitialUserType>(state => state.common.currentUser);
```

Comment:
```suggestion
  const { first_name, second_name } = useAppSelector(state => state.common.currentUser);
```

`<InitialUserType>` - это лишнее, так как и так по селектору будет вычислен тип

Reply:
Done